app/models/attendance.rb

```diff
@@ -9,6 +9,8 @@
 #  updated_at  :datetime         not null
 #  employee_id :bigint           not null
 #
+# frozen_string_literal: true
+
```

Comment:
lets try to add this magic comment at the beginning of the file.

Reply:
Ok ok thanks!